### ISSUE TITLE
Caraboidea is not under Coleoptera

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#460]
classified directly under Insects instead in AC22

**Comment by mdoering** [comment]
https://www.checklistbank.org/dataset/9837/taxon/4WK

**Comment by yroskov** [comment]
Yeah, a bug. It was caused by this problem: https://github.com/CatalogueOfLife/backend/issues/1042
